test/proposal_subvol_test.rb

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env rspec
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require_relative "spec_helper"
+require "storage"
+require "y2storage"
+require_relative "support/proposal_examples"
+require_relative "support/proposal_context"
+
+describe Y2Storage::Proposal do
+  describe "#propose subvolumes" do
+    include_context "proposal"
+
+    let(:subvol) { true }
+
+    let(:proposal_devicegraph_yaml) do
+      proposal.propose
+      yaml_writer = Y2Storage::YamlWriter.new
+      yaml_writer.yaml_device_tree(proposal.devices.to_storage_value)
+    end
+
+    let(:root_yaml) do
+      disk = proposal_devicegraph_yaml.first["disk"]
+      partitions = disk["partitions"]
+      partitions.map do |pslot|
+        part = pslot["partition"]
+        next nil unless part
+        part["mount_point"] == "/" ? part : nil
+      end.compact.first
+    end
+
+    let(:subvol_yaml) do
+      root = root_yaml || {}
+      btrfs_yaml = root["btrfs"] || {}
+      btrfs_yaml["subvolumes"] || []
+    end
+
+    # rubocop:disable Metrics/LineLength
```

Comment:
Rubocop disables ones again. Maybe in this particular case is not that bad (although we have managed to write many tests without disabling it), but I'm worried about the trend.